experiment/test/data/make_rcm_data.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env python
+"""
+This script auto-generates a sample on-disk dataset for testing.
+
+"""
+
+import numpy as np
+import os
+import pandas as pd
+import xarray as xr
+
+from experiment import Experiment, Case
+
+PATH_TO_DATA = os.path.join(os.path.dirname(__file__), "RCM data")
+SEPARATOR = "_"
+cases = [
+    Case("category", "Climate Data Category", ["Temperature Climate Data"]),
+    Case("variable", "Climate Variable", ["Temperature",
```

Comment:
This deserves an offline discussion. Usually when I use `experiment`, I don't include the variable or "field" as part of the path;  I consider those output files to be the leaves of the file system/tree I'm trying to parse. I guess it becomes important if the field name is in the experiment hierarchy... but maybe that would be a worthy, separate change to the package to have an option to inject the field where it comes up.